app/assets/stylesheets/views/_covid.scss

```diff
@@ -125,6 +125,19 @@
     margin-bottom: govuk-spacing(4);
   }
 }
+.covid__spaced-list>li {
```

Comment:
I'm late to this party but wanted to point out that we always should be applying styles using classes, not element selectors such as `li` here.